fe/lib/components/JobCategorySuggest/JobCategorySuggestChoices.tsx

```diff
@@ -96,6 +101,24 @@ const JobCategorySuggestChoices = forwardRef<HTMLInputElement, Props>(
             );
           })}
         </RadioGroup>
+        {selectedJobCategory === 'Other' && (
+          <JobCategorySelect
+            client={client}
+            name={name}
+            id="jobCategoryId"
+            onSelect={(jobCategory: JobCategoryAttributesFragment) => {
```

Comment:
Is this type annotation required? Ideally the `JobCategorySelect` component would have its types sorted out itself.